source/darwin/darwin_pki_utils.c

```diff
@@ -180,7 +211,7 @@ int aws_import_public_and_private_keys_to_identity(
     return result;
 }
 
-#endif /* AWS_OS_IOS */
+#endif /* !AWS_OS_IOS */
```

Comment:
nit: typo?

Reply:
The if statement is actually `#if !defined(AWS_OS_IOS)`, so added the `!` to comments avoid confusing.